apps/studio/components/interfaces/DataWarehouse/CreateWarehouseAccessToken.tsx

```diff
@@ -0,0 +1,69 @@
+import { set } from 'lodash'
+import React from 'react'
```

Comment:
for consistency: could we just import the things that we need from react?

e.g `import { useState } from 'react'`

Reply:
similar for the other files